invenio_jobs/assets/semantic-ui/js/invenio_jobs/administration/RunActionForm.js

```diff
@@ -151,6 +162,11 @@ export class RunActionForm extends Component {
                   {!isEmpty(error) && (
                     <ErrorMessage
                       {...error}
+                      content={
+                        actions_errors && Object.keys(actions_errors).length > 0
```

Comment:
Added this in order to be able to display the exact error message of a marshmallow validation in the UI. Here is gets populated: https://github.com/inveniosoftware/invenio-administration/blob/main/invenio_administration/assets/semantic-ui/js/invenio_administration/src/formik/ActionForm.js#L119

Before: 
![Screenshot 2025-02-07 at 16 25 52](https://github.com/user-attachments/assets/8cce6fe7-c022-4d56-8b2c-cf4f0af241a2)

After: 
![Screenshot 2025-02-07 at 16 25 09](https://github.com/user-attachments/assets/c5fd90b9-6bf6-4df3-97d1-3f000c60d217)